docs/source/demos/lead_scoring.ipynb

```diff
@@ -286,12 +293,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When we optimize for AUC, we can see that the AUC score from this pipeline is better than the AUC score from the pipeline optimized for lead scoring. However, the revenue per lead gained was only `$7` per lead when optimized for AUC and was `$45` when optimized for lead scoring. As a result, we would gain up to 6x the amount of revenue if we optimized for lead scoring.\n",
+    "When we optimize for AUC, we can see that the AUC score from this pipeline is better than the AUC score from the pipeline optimized for lead scoring. However, the revenue per lead suffered a loss of `$0.03` per lead when optimized for AUC and was `$14.6` when optimized for lead scoring. As a result, we would have a huge gain on the amount of revenue if we optimized for lead scoring.\n",
```

Comment:
I think this is another place where we should de-quantify the notebook prose and word it in a more general way.